front/src/components/ArticleSendCopy.jsx

```diff
@@ -23,13 +25,13 @@ export default function ArticleSendCopy ({ article }) {
           variables: {user: null, to: user._id, article: article._id}
         })
         setToast({
-          text: `Copie de l'article envoyée à ${user.displayName || user.username}.`,
+          text: t('article.sendCopy.success') ` ${user.displayName || user.username}.`,
```

Comment:
soit on utilise `notification` partout quand le message est utilisé dans in toast soit on ne le précise pas